test/intl402/Intl/getCanonicalLocales/main.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 Mozilla Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+esid: sec-intl.getcanonicallocales
+description: Tests for existance and behavior of Intl.getCanonicalLocales
+info: |
+  8.2.1 Intl.getCanonicalLocales (locales)
+  1. Let ll be ? CanonicalizeLocaleList(locales).
+  2. Return CreateArrayFromList(ll).
+includes: [compareArray.js]
+---*/
+
+if (typeof Intl.getCanonicalLocales !== "function") {
+    $ERROR("Intl.getCanonicalLocales doesn't exist.");
+}
+
+if (Intl.getCanonicalLocales.length !== 1) {
+    $ERROR("Intl.getCanonicalLocales.length should be 1.");
+}
```

Comment:
we have a non-explicit pattern for existence that can be split in 3 test files:

Those files would be:

- property type (function) and its descriptor. [example](https://github.com/tc39/test262/blob/master/test/built-ins/Array/prototype/find/find.js)
- length's value, and descriptor. [example](https://github.com/tc39/test262/blob/master/test/built-ins/Array/prototype/find/length.js)
- name's value and descriptor. [example](https://github.com/tc39/test262/blob/master/test/built-ins/Array/prototype/find/name.js)